tools/migration/cfg/migrate.py

```diff
@@ -24,9 +25,12 @@ def main():
     input_version =  utils.get_conf_version(args.input_path)
     curr_dir = os.path.dirname(__file__)
     chain = []
+    if input_version == target_version:
+        print ("version of input cfg is identical to target:", input_version)
```

Comment:
suggestion: "The target version and source version in harbor.cfg  is the same: %s. No need to upgrade.", input_version

Reply:
Will remove this since we'll handle the `==` case in `search`